nntrainer/meson.build

```diff
@@ -101,9 +105,13 @@ else
     install_dir: nntrainer_libdir
   )
 
-  nntrainer_lib = nntrainer_shared
-  if get_option('default_library') == 'static'
+  if host_machine.system() == 'windows'
```

Comment:
Comment? What is going on here?

Reply:
Maybe we should ban `default_library=shared` build on windows, or override it in native file and warn/error at toplevel